elastic/beats
# filestream: fix 'requires pointer' error

## What does this PR do?

This PR fixes the 'requires pointer' error log issued by filestream when a file being harvested is renamed.

Filebeat already recovers from this error by using the file indentifier from the prospector instead of the metadata, so the log message is downgraded to warn and contains more details about what happened.


## Why is it important?

It fixes a bug and removes a log error that was somehow misleading.

## Checklist
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding change to the default configuration files~~

~~## Author's Checklist~~
## How to test this PR locally
1. Setup filestream to harvest any file
2. Rename the file
3. You should not see any log error

~~## Related issues~~
~~## Use cases~~
~~## Screenshots~~
~~## Logs~~
